stdlib/Test/test/runtests.jl

```diff
@@ -584,7 +584,7 @@ end
         @test total_error  == 6
         @test total_broken == 0
     end
-    ts.anynonpass = false
+    @atomic ts.anynonpass = false
```

Comment:
maybe switch to 0x0? to reflect the new type (or an enum)

Reply:
Yeah, I'll switch to 0x00. This `anynonpass` thing is a hack anyway and would be better put in a printing state object, so I don't think it's worth putting an enum in for it, but I didn't want to do that larger refactor at this point.